layouts/home.html

```diff
@@ -72,111 +72,108 @@ <h1 class="text-5xl font-medium">{{ .Title }}</h1>
+          )  
+	) }}
         {{ partial "docs-section.html" (dict
-          "Title" "Redis Data Integration"
-          "Description" "Redis Data Integration keeps Redis in sync with the primary database in near real time."
-          "ButtonLink" "./integrate/redis-data-integration"
-          "ButtonLabel" "Learn more"
-          "LinksLeftTitle" "LEARN MORE"
+         "Title" "Optimize Redis for high-scale apps"
+          "Description" "Get the Redis Insight GUI on AWS EC2, Docker, Kubernetes, and desktop"
+	  "ButtonLink" "./operate/redisinsight/install/"
+          "ButtonLabel" "Redis Insight installation guide"  
           "LinksLeft" (slice
-            (dict "Text" "Quick start" "URL" "./integrate/redis-data-integration/quick-start-guide")
-            (dict "Text" "Architecture" "URL" "./integrate/redis-data-integration/architecture")
-            (dict "Text" "Install" "URL" "./integrate/redis-data-integration/installation")
-            (dict "Text" "Reference" "URL" "./integrate/redis-data-integration/reference")
+            (dict "Text" "Performance and memory optimization user guide" "URL" "./develop/tools/insight/#database-analysis")
           )
         ) }}
         {{ partial "docs-section.html" (dict
-          "Title" "Redis Insight"
-          "Description" "A cross-platform GUI for Redis, with focus on reducing memory usage and improving application performance."
-          "ButtonLink" "./develop/tools/insight/"
-          "ButtonLabel" "Learn more"
-          "LinksLeftTitle" "Learn more"
+         "Title" "Migrate data to or from Redis"
+          "Description" "Migrate data from files, data generators, relational databases, or snapshot replication"
+	  "ButtonLink" "./integrate/riot/"
+          "ButtonLabel" "Redis input and output CLI tool user guide"  
           "LinksLeft" (slice
-            (dict "Text" "Install" "URL" "./operate/redisinsight/install/")
-            (dict "Text" "User guide" "URL" "./develop/tools/insight/")
-            (dict "Text" "Manage Streams in Redis Insight" "URL" "./develop/tools/insight/tutorials/insight-stream-consumer/")
+            (dict "Text" "Redis Input/Output Tool quickstart guide" "URL" "./integrate/riot/quick-start/")
           )
         ) }}
+	{{ partial "docs-section.html" (dict
+	  "Title" "Connect with Redis client API libraries"
+          "Description" "Connect your application to a Redis database and try an example"
+          "ButtonLink" "./develop/clients/"
+          "ButtonLabel" "Redis client API library guides"
+       
+          "LinksLeft" (slice
+            (dict "Text" "Python" "URL" "./develop/clients/redis-py/")
+            (dict "Text" "Python (vector)" "URL" "./integrate/redisvl/")
+	    (dict "Text" "C#/.NET" "URL" "./develop/clients/dotnet/")
+	    (dict "Text" "JavaScript" "URL" "./develop/clients/nodejs/")
+            (dict "Text" "Java (synchronous)" "URL" "./develop/clients/jedis/")
```

Comment:
For this type of quick entry point thing, I'd suggest leaving out Python vector and leave the "synchronous" bit off Java. RedisVL and Lettuce are a bit niche compared to the main libraries and they are easy enough to find (although we should add a link from the redis-py page to RedisVL). This also leaves some room on the tile to add Go and PHP. See also my comment about the Client Tools tile below.
```suggestion
            (dict "Text" "Python" "URL" "./develop/clients/redis-py/")
	    (dict "Text" "C#/.NET" "URL" "./develop/clients/dotnet/")
	    (dict "Text" "JavaScript" "URL" "./develop/clients/nodejs/")
            (dict "Text" "Java" "URL" "./develop/clients/jedis/")
            (dict "Text" "Go" "URL" "./develop/clients/go/")
            (dict "Text" "PHP" "URL" "./develop/clients/php/")
```